mappings/dependencies.json

```diff
@@ -247,6 +247,19 @@
         "Xamarin.AndroidX.Lifecycle.Common"
       ]
     },
+    {
+      "id": "Xamarin.AndroidX.Car.App",
```

Comment:
Should this be `Xamarin.AndroidX.Car.App.App`?

Reply:
true. Let me rerun local builds one more time. Not sure why this was not updated.